protostar/commands/test/test_results.py

```diff
@@ -78,6 +78,23 @@ def from_failed_test_case_result(
         )
 
 
+@dataclass(frozen=True)
+class SetupCaseResult(TestResult, TimedTestResult):
```

Comment:
- [ ] `SetupCaseResult` sounds weird. Wouldn't explicit form (`SetupTestCaseResult`) be better?

Reply:
I'm not really keen to rename this to `SetupTestCaseResult` because it sounds broken English to me. This can be named `TestCaseSetupResult`, but in general this contradicts the `setup case` terminology I use everywhere.

I'm open to renaming the concept of `setup case` to something else, like `test case setup hook` (though this one is loong). Just need ideas :)

Reply:
`SetupTestResult`?

Anyway, let's stick with the setup case as long as it used consistently.